CHANGELOG.rst

```diff
@@ -13,6 +13,7 @@ History
 
 * Removed ``user_defined_esp_table`` and ``esp_parameters`` from ``PumpEquipmentDescription``.
 
+* Added ``pyinvoke`` support.
```

Comment:
"support" conveys the idea that is somehow optional, but is in fact central to how we manage tasks now.

```suggestion
* ``pyinvoke`` is now used to manage plugins tasks.
```

Reply:
Fixed